### PR TITLE
Update copy for Backup product cards

### PR DESCRIPTION
**Pull request by monsieur-z**
### Changes proposed in this Pull Request

This PR includes the final copy for the Jetpack Backup, Backup _Daily_, and Backup _Real-time_ product cards.

Jetpack Complete was handled in https://github.com/Automattic/wp-calypso/pull/45009.
Security in https://github.com/Automattic/wp-calypso/pull/45035.

### Implementation notes

For the activity log feature item, I used the copy already used for Jetpack Complete and Security.

### Testing instructions

- Visit the _Plans_ page with the offer reset flow enabled
- Check the copy of the Jetpack Backup product card against the copydeck
- Don't forget to check the descriptions and links in the tooltips
- Click on "Get Backup"
- Check the copy of the Backup _Daily_ and _Real-time_ cards against the copydeck

### Screenshots

Jetpack Backup
Screen Shot 2020-08-19 at 4 48 37 PM

Backup options
Screen Shot 2020-08-19 at 4 33 42 PM

### CODE REVIEW

File: client/lib/products-values/translations.js [hunk 1]
```diff
@@ -114,16 +114,16 @@ export const getJetpackProductsDescriptions = () => {
 	);
 	return {
 		[ CONSTANTS.PRODUCT_JETPACK_BACKUP_DAILY ]: translate(
-			'Always-on backups ensure you never lose your site. Your changes are saved every day with a 30-day archive.'
+			'Never lose a word, image, page, or time worrying about your site'
```
Comment (rcanepa) [inline client/lib/products-values/translations.js:117]: I think we need a `.` at the end of the sentence here, right after `'... about your site'`.

File: client/lib/products-values/translations.js [hunk 2]
```diff
@@ -114,16 +114,16 @@ export const getJetpackProductsDescriptions = () => {
...
 		),
 		[ CONSTANTS.PRODUCT_JETPACK_BACKUP_DAILY_MONTHLY ]: translate(
-			'Always-on backups ensure you never lose your site. Your changes are saved every day with a 30-day archive.'
+			'Never lose a word, image, page, or time worrying about your site'
```
Comment (rcanepa) [inline client/lib/products-values/translations.js:120]: Same here

File: client/lib/products-values/translations.js (hunk 1 above)
Comment (monsieur-z) [reply client/lib/products-values/translations.js:117]: Absolutely. I noticed it afterwards and fixed it in https://github.com/Automattic/wp-calypso/pull/45067.

**Comment by monsieur-z** [comment]
Absolutely. I noticed them afterwards and fixed it in https://github.com/Automattic/wp-calypso/pull/45067.

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/4247509

Thank you @monsieur-z for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/4247509).

**Pull request merged**